src/components/NavBar.tsx

```diff
@@ -1,15 +1,59 @@
 import React from 'react';
-import { AppBar, Tabs, Tab } from '@material-ui/core';
-import { Link } from 'react-router-dom';
+import { useHistory, useLocation } from 'react-router-dom';
+import { Layout, Menu } from 'antd';
+const { Header, Footer, Sider, Content } = Layout;
 
 const NavBar: React.FC = () => {
+  const history = useHistory();
+  const location = useLocation();
+  const path: string = location.pathname;
+
+  function pathnameToKey(pathname: string) {
+    switch (pathname) {
+      case '/':
+        return '1';
+      case '/block-template':
+        return '2';
+      case '/grid-template':
+        return '3';
+      default:
+        return '1';
+    }
+  }
+
   return (
-    <AppBar position="static" color="transparent">
-      <Tabs value={false} variant="scrollable" scrollButtons="auto">
-        <Tab label="Home" component={Link} to="/" />
-        <Tab label="Other Example Page" component={Link} to="/example" />
-      </Tabs>
-    </AppBar>
+    <Header>
+      <Menu
+        theme="dark"
+        mode="horizontal"
+        defaultSelectedKeys={[pathnameToKey(path)]}
+      >
+        <Menu.Item
+          key="1"
+          onClick={() => {
```

Comment:
Hmm, I think there's a linting rule in TS against inline lambdas for performance reasons, so we may want to consider another approach/enable that. 
https://jonhilton.net/typescript-and-react-forbidden-lambdas/

Reply:
Hmm ok, I think its less expressive though.